pkg/ruler/ruler.go

```diff
@@ -36,7 +36,7 @@ var (
 		Namespace: "cortex",
 		Name:      "group_evaluation_duration_seconds",
 		Help:      "The duration for a rule group to execute.",
-		Buckets:   []float64{.1, .25, .5, 1, 2.5, 5, 10, 25},
+		Buckets:   []float64{.5, 1, 2.5, 5, 10, 25, 60, 120},
```

Comment:
This change isn't mentioned in the commit message.

Reply:
I'll will take this out 👍 

I have some rule groups that take a long time to eval and having larger buckets can be nice